base/changes.txt

```diff
@@ -4,6 +4,17 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 =======================================================================
 
+#########################
+# 2020-02-07 PL X Release
+#########################
+
+2020-02-07  Will Robertson  <will.robertson@latex-project.org>
+
+	* ltfssini.dtx:
+	XeTeX-specific definition for \symbol to avoid bug in maths mode (gh/124)
```

Comment:
Is there any reason why this is specifically a bug in math mode? If I'm not mistaken the same issue should appear in other modes.

Reply:
No I haven't tried to trace the issue in the engine. Long story short, `\char"1D400\relax` is fine in text mode but fails in maths mode.